packages/extension-browser/tests/content-script.ts

```diff
@@ -0,0 +1,221 @@
+import { JSDOM } from 'jsdom';
+import * as proxyquire from 'proxyquire';
+import * as sinon from 'sinon';
+import test from 'ava';
+
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { FetchEnd } from 'hint/dist/src/lib/types';
+
+import { Config, Events, Results } from '../src/shared/types';
+
+import { readFixture } from './utils/helpers';
+
+const base = '../src/content-script';
+const paths: { [name: string]: string } = {
+    'web-async-html': `${base}/web-async-html`,
+    connector: `${base}/connector`, // eslint-disable-line
```

Comment:
What is `eslint` complaining about in here?

Reply:
The order of the properties not being alphabetical. However they need to be in this order as the lowest dependency must be overridden before modules which depend on it. I added a comment to clarify.